changelogs/fragments/8738-limit-packages-for-copr.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - copr - Added `includepkgs` parameter to filter packages from the repository (https://github.com/ansible-collections/community.general/pull/8779).
```

This is later parsed into RST, so keywords like that need two backticks on each side.
```suggestion
  - copr - Added ``includepkgs`` parameter to filter packages from the repository (https://github.com/ansible-collections/community.general/pull/8779).
```

---

thanks for the advice , updated the change as asked